cornflow/schemas/case.py

```diff
@@ -1,5 +1,7 @@
 from marshmallow import fields, Schema
 
+from .common import QueryFilters as QueryFiltersCase
+
 
```

Comment:
so it doesn't appear it's not being used I edited this line in the instance and execution schema so it looks like this:

```
class QueryFiltersCase(QueryFilters)
    pass
```